src/transformers/agents/agent_types.py

```diff
@@ -88,7 +88,8 @@ class AgentImage(AgentType, ImageType):
     """
 
     def __init__(self, value):
-        super().__init__(value)
+        AgentType.__init__(self, value)
+        ImageType.__init__(self)
```

Comment:
I found that without this, the `ImageType` attributes would not be inherited.

Reply:
Hmmm I'm curious why that is the case, I'll investigate